scripts/deployment/main.py

```diff
@@ -196,56 +209,39 @@ def main(
     password: str,
     output_dir: Path,
     config_file: Path,
-    test_messenger: bool,
     allow_same_chain: bool,
 ) -> None:
-    commit_id = get_commit_id()
     account = account_from_keyfile(keystore_file, password)
+    print("Deployer:", account.address)
```

Comment:
This should probably go away

Reply:
Actually, I added this intentionally since it proved useful when I was inspecting the output of the deployment script and correlating that with the E2E script. It may come in handy so I'd prefer to leave it in, but if you feel strongly about it, I can remove it.